library/ssl_tls.c

```diff
@@ -58,6 +58,54 @@
 
 #define PROPER_HS_FRAGMENT 0x75555555
```

Comment:
@AndrzejKurek , lets make this value 2 bytes? It will reduce 25 bytes of code :)

Reply:
I tested it and it's 16 bytes, but sure, I'll add it :)